src/versions.js

```diff
@@ -2,37 +2,11 @@ const versions = {
     'react-cm-ui': {
         package: '9.21.4',
         components: {
-            dataDisplay: {
-                dataCard: {
-                    devLibraryVersion: 'N/A',
-                    designLibraryVersion: 'N/A',
-                    designLibraryDoc: 'N/A',
-                },
-            },
-            inputs: {
-                durationPicker: {
+            layout: {
+                grid: {
                     devLibraryVersion: '2.0.0',
-                    designLibraryVersion: '', // TODO: Add Design Library version
-                    designLibraryDoc: '', // TODO: Add Sketch Cloud Link
-                },
-            },
-            navigation: {
-                breadcrumbs: {
-                    devLibraryVersion: '2.0.1',
-                    designLibraryVersion: '', // TODO: Add Design Library version
-                    designLibraryDoc: '', // TODO: Add Sketch Cloud Link
-                },
-                sectionalTabs: {
-                    devLibraryVersion: '2.0.1',
-                    designLibraryVersion: '', // TODO: Add Design Library version
-                    designLibraryDoc: '', // TODO: Add Sketch Cloud Link
-                },
-            },
-            surfaces: {
-                appBar: {
-                    devLibraryVersion: '1.0.0',
-                    designLibraryVersion: '', // TODO: Add Design Library version
-                    designLibraryDoc: '', // TODO: Add Sketch Cloud Link
```

Comment:
Deliberately deleting all of these? 😕

Reply:
Yes sir, these were all meaningless. I plan to put in real version numbers and links to update components in 10.0.0.

Reply:
Okay, I trust you.  Just checking.
Resolving.